src/components/List.jsx

```diff
@@ -1,23 +1,54 @@
 import React from 'react';
 import db from '../lib/firebase';
-import { collection } from 'firebase/firestore';
+import { collection, doc, updateDoc } from 'firebase/firestore';
 import { useCollection } from 'react-firebase-hooks/firestore';
+import moment from 'moment';
 
 export const List = ({ token }) => {
   const [value, loading, error] = useCollection(collection(db, token), {
     snapshotListenOptions: { includeMetadataChanges: true },
   });
 
+  const calcTimeDiff = (purchasedTime) => {
+    let timeNow = moment().format();
+    var date1 = moment(timeNow, 'YYYYMMDD HH:mm:ss');
+    var date2 = moment(purchasedTime, 'YYYYMMDD HH:mm:ss');
+    let timeDiff = date1.diff(date2, 'hours');
```

Comment:
I recommend using either `let` or `const` instead of `var`. It's a good practice to use `const` so that you don't mutate data unless it is intentional. 
```suggestion
    const timeNow = moment().format();
    const date1 = moment(timeNow, 'YYYYMMDD HH:mm:ss');
    const date2 = moment(purchasedTime, 'YYYYMMDD HH:mm:ss');
    const timeDiff = date1.diff(date2, 'hours');
```

Reply:
Thanks for catching that Yenly. Update the variables to const. 😊